examples/stable/test/property/canister_methods/on_low_wasm_memory/src/index.ts

```diff
@@ -0,0 +1,73 @@
+import { call, IDL, onLowWasmMemory, query, update } from 'azle';
+
+export default class MemoryManagementCanister {
+    storedBytes: Uint8Array = new Uint8Array(0);
+    hasLowMemoryHandlerBeenCalled: boolean = false;
+
+    @onLowWasmMemory
+    onLowWasmMemory(): void {
+        this.hasLowMemoryHandlerBeenCalled = true;
+        this.storedBytes = new Uint8Array(0);
+    }
+
+    @query([], IDL.Bool)
+    wasLowMemoryHandlerCalled(): boolean {
+        return this.hasLowMemoryHandlerBeenCalled;
+    }
+
+    @update([], IDL.Vec(IDL.Nat8))
+    clearBytes(): Uint8Array {
+        this.storedBytes = new Uint8Array(0);
+        return new Uint8Array(this.storedBytes);
+    }
+
+    @query([], IDL.Nat32)
+    getStoredByteCount(): number {
+        return this.storedBytes.length;
+    }
+
+    @update([IDL.Nat32])
+    async addRandomBytes(bytesToAdd: number): Promise<void> {
+        const randomBytes = await fetchRandomBytesFromIC();
+        const expandedBytes = expandRandomBytes(randomBytes, bytesToAdd);
+
+        const newBuffer = new Uint8Array(
```

Comment:
Simplify this concatenation to the simplest possible simplification.

Reply:
That's it, it was essentially already there